build/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.10.0
```

Comment:
I personally like adding the -dev here too until we do a release, but up to you.